docs/using.rst

```diff
@@ -538,8 +538,14 @@ commands into your individual environment.
 Modifying the Renewal Configuration File
 ----------------------------------------
 
+When a certificate is issued Certbot creates a renewal configuration file that
```

Comment:
nit: This isn't always true. If you create a certificate with a custom CSR using `--csr`, a renewal conf file isn't created. I have a slight preference towards mentioning this somewhere here for correctness and to avoid confusion for the few `--csr` users out there, but I understand if you want to leave this out to simplify our documentation. I wanted to make sure you were aware of this caveat though.